tools/chocolateyinstall.ps1

```diff
@@ -9,11 +9,11 @@ $packageArgs = @{
   specificFolder  = 'lite-xl'
 
   url             = 'https://github.com/lite-xl/lite-xl/releases/download/v2.0.3/lite-xl-windows-x86.zip'
```

Comment:
You need to update this URL as well because it has the version in it. Alternatively this could be templated at some point.

Reply:
i fixed it (at least i hope i did)